uiFlows/login_flow.yaml

```diff
@@ -0,0 +1,18 @@
+appId: org.simple.clinic.staging
+---
+- clearState
+- launchApp
+- tapOn: "Next"
+- tapOn: "Get started"
+- tapOn:
+    text: "India"
+- tapOn:
+    text: "Maharashtra"
```

Comment:
I don't see Maharashtra in my demo app selection screen. Is there a better way of handling these clicks events instead of relying on data pushed from server? 🤔